<?xml version="1.0"?>

<launch>
  <arg name="use_dji" default="True" />
  <arg name="headless" default="False" />
  <node pkg="jsk_uav_forest_common" type="tf_broadcaster.py" name="tf_broadcaster" output="screen" >
    <param name="uav_odom_topic_name" value="/modified_odom" if="$(arg use_dji)" />
    <param name="uav_odom_topic_name" value="/ground_truth/state" unless="$(arg use_dji)" />
  </node>

  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find jsk_uav_forest_common)/launch/log_viewer.rviz" unless="$(arg headless)" respawn="true"/>

</launch>
